src/Libraries/DSIronPython/IronPythonEvaluator.cs

```diff
@@ -33,30 +33,35 @@ public static class IronPythonEvaluator
         private static string prev_code { get; set; }
         /// <summary> stores a copy of the previously compiled engine</summary>
         private static ScriptSource prev_script { get; set; }
-        /// <summary> stores a reference to the executing Dynamo Core directory</summary>
-        private static string dynamoCorePath { get; set; }
+
+        /// <summary> stores a reference to path of IronPython std lib</summary>
+        private static string pythonLibDir { get; set; }
```

Comment:
To save time in multiple evaluations, I am storing this path. I think it will be much more efficient than storing the old execution path here.